src/test.ts

```diff
@@ -122,20 +157,40 @@ export function consoleTestResultHandler(testResult: TestResult): boolean {
         const results = testResult.results[fileName];
         process.stdout.write(`${fileName}:`);
 
-        const diffResults = diff.diffLines(results.markupFromMarkup, results.markupFromLinter);
-        const didTestPass = !diffResults.some((diff) => diff.added || diff.removed);
+        const markupDiffResults = diff.diffLines(results.markupFromMarkup, results.markupFromLinter);
+        const fixesDiffResults = diff.diffLines(results.fixesFromMarkup, results.fixesFromLinter);
+        const didMarkupTestPass = !markupDiffResults.some((diff) => diff.added || diff.removed);
+        const didFixesTestPass = !fixesDiffResults.some((diff) => diff.added || diff.removed);
 
         /* tslint:disable:no-console */
-        if (didTestPass) {
+        if (didMarkupTestPass && didFixesTestPass) {
             console.log(colors.green(" Passed"));
         } else {
             console.log(colors.red(" Failed!"));
-            console.log(colors.green(`Expected (from ${FILE_EXTENSION} file)`));
+        }
+        if (!didMarkupTestPass) {
+            console.log(colors.green(`Expected markup (from ${MARKUP_FILE_EXTENSION} file)`));
+            console.log(colors.red("Actual (from TSLint)"));
+
+            didAllTestsPass = false;
+
+            for (const diffResult of markupDiffResults) {
```

Comment:
consider extracting a helper function to print the failure, rather than duplicate this between markup failure and diff failure?